setup.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+import subprocess
+from distutils.core import setup
+
+git_version = 'UNKNOWN'
```

Comment:
No need for the git version malarkey imho, just `version='0.0.0'` in the `setup()` invocation.